### PR TITLE
Task3

### PR DESCRIPTION
**Pull request by OlgaBelyakovich**
hope everything is ok

### CODE REVIEW

File: task1/task3/Program.cs [hunk 1]
```diff
@@ -0,0 +1,46 @@
...
+    internal class Program
+    {
+        static void Main(string[] args)
+        {
+            //input five numbers and output ten words, each work being a "True" or a "False". the i-th word
+            //(1-based (it means, the first word is considered to have an index 1 here)) should be true if i appears among the numbers entered;
+            //otherwise, the i-th word should be false. for example, if a user inputs (in 5 lines): 1 6 7 101 22,
+            //the output should be True False False False False True True False False False (10 lines).
+
+            bool [] arrValue = new bool [11];
```
Comment (dartenby) [inline task1/task3/Program.cs:18]: why 11? needs to be 10

File: task1/task3/Program.cs [hunk 2]
```diff
@@ -0,0 +1,46 @@
...
+
+            for (int i = 0; i < arrNum.Length; i++)
+            {
+                count++;
+                Console.WriteLine($"Enter the {count} number:");
+                while (!int.TryParse(Console.ReadLine(), out arrNum[i]))
+                {
+                    Console.WriteLine("Enter the correct number.");
+                }
+                for (int j = 1; j < arrValue.Length; j++)
```
Comment (dartenby) [inline task1/task3/Program.cs:31]: needs to be 0

File: task1/task3/Program.cs [hunk 3]
```diff
@@ -0,0 +1,46 @@
...
+                {
+                    if (arrNum[i] == j)
+                    {
+                        arrValue[j] = true;
+                    }
+                }
+            }
+
+            for (int j = 1; j < arrValue.Length; j++)
```
Comment (dartenby) [inline task1/task3/Program.cs:40]: 0

File: task1/task3/Program.cs (hunk 1 above)
Comment (OlgaBelyakovich) [reply task1/task3/Program.cs]: fixed

File: task1/task3/Program.cs (hunk 2 above)
Comment (OlgaBelyakovich) [reply task1/task3/Program.cs]: fixed

File: task1/task3/Program.cs (hunk 3 above)
Comment (OlgaBelyakovich) [reply task1/task3/Program.cs]: fixed

**Pull request merged**